test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -81,6 +81,8 @@ public void TestTextLoaderDataTypes()
 
                 Assert.Equal(i, sByteTargets.Length);
             }
+
+            Done();
```

Comment:
💭 I thought we were going to move this call to the base test class?

Reply:
Yes, ideally but that should be other PR, this PR is intended to fix these few tests and make proper assertion

---
In reply to: [375545013](https://github.com/dotnet/machinelearning/pull/4782#discussion_r375545013) [](ancestors = 375545013)